packages/cli/README.md

```diff
@@ -37,9 +42,29 @@ npm run extract
 npm run compile
 ```
 
+### 2) Use NPX
+
+You can run the scripts directly using a tool for executing Node packages `NPX`. `NPX` is included in `NPM` version 5.2 and higher.
+
+```
+npx lingui add-locale -- en cs
```

Comment:
Add locale doesnt exist anymore

Reply:
I am also looking into https://github.com/lingui/js-lingui/blob/main/packages/cli/src/api/types.ts and it looks to me that most types are not used. Should I remove them?

Reply:
For example what is a purpose of this `CatalogApi` export https://github.com/lingui/js-lingui/blob/main/packages/cli/src/api/types.ts#L58 Isn't it better to define it directly in catalog.ts file?